docs/using.rst

```diff
@@ -67,7 +67,7 @@ contain useful documentation and a quickstart guide.
   * `Vagrantfile <../components/centos/centos-docker-base-setup/Vagrantfile>`_
   * `README <../components/centos/centos-docker-base-setup/README.rst>`_
 
-* Docker and Kubernetes for use with host-based tools or via ``vagrant ssh``
+* Kubernetes for use with host-based tools or via ``vagrant ssh``
```

Comment:
@Preeticp Could you step 2 from this section https://github.com/Preeticp/adb-atomic-developer-bundle/blob/1380501b507cc1e54732e5468163e3d34e74b874/docs/using.rst#starting-the-vagrant-box

As it is not required since now private network is being automatically set by vagrant service manager for virtualbox.